src/relay/backend/compile_engine.cc

```diff
@@ -770,7 +770,8 @@ class CompileEngineImpl : public CompileEngineNode {
       With<PassContext> fresh_pass_ctx_scope(PassContext::Create());
 
       std::unordered_map<te::Tensor, tir::Buffer> binds;
-      cache_node->funcs = tvm::lower(cfunc->schedule, all_args, cache_node->func_name, binds);
+      cache_node->funcs =
+          tvm::LowerSchedule(cfunc->schedule, all_args, cache_node->func_name, binds);
```

Comment:
We shouldn't need to route through the `relay.backend.lower` packed function any longer right? I'd suggest either removing it and the above conditional (line 766) or also reproducing it's functionality in c++ if we do need it as part of this PR. This way we avoid an unnecessary round trip through python.

Reply:
Yeah I don't think we do. I'll remove it (I didn't look too closely at what this call was doing, oops!)